src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java

```diff
@@ -119,11 +124,13 @@ private int outputTest(StringBuilder buffer, TestResult failedTest,
         local.append('.').append(failedTest.getDisplayName()).append('\n');
 
         if (showMessage) {
-            local.append("\nError Message:\n").append(failedTest.getErrorDetails()).append('\n');
+            String errorDetails = escapeHtml ? escapeHtml(failedTest.getErrorDetails()) : failedTest.getErrorDetails();
+            local.append("\nError Message:\n").append(errorDetails).append('\n');
         }
-        
+
         if (showStack) {
-            local.append("\nStack Trace:\n").append(failedTest.getErrorStackTrace()).append('\n');
+            String stackTrace = escapeHtml ? escapeHtml(failedTest.getErrorStackTrace()) : failedTest.getErrorStackTrace();
+            local.append("\nStack Trace:\n").append(stackTrace).append('\n');

```

Comment:
When using escapeHtml, would you want to use `<br/>` instead of `\n` in the strings?

Reply:
Good point, I've changed it.